tests/python/op/test_nn.py

```diff
@@ -101,5 +101,43 @@ def forward(self, x_input):
     verify_step(Model(), [x], jit_script=False)
 
 
+# @pytest.mark.parametrize("dtype", [torch.float32, torch.float16])
```

Comment:
why it is commented out? Also we can just have two cases for `p` to reduce the test number